### DISCUSSION TITLE
Masa Mobile Navigation not displaying

### DISCUSSION THREAD

**Discussion by pshaughnessy88** [MasaCMS/MasaCMS#234] [Q&A]
Hello,

We are working on a website redesign for our library. I had played around with Mura 7.1 a few years ago and want to try out a Masa sandbox. Last week we installed version 7.4.3 and found while the navigation works perfectly on the desktop, for some reason it is not toggling to mobile on a phone.

I upgraded to the Bootstrap 5 theme and followed the steps listed in issue #108. Unfortunately it is still not working on mobile.

Does anyone have suggestions on how we can fix this?

Thanks,
Peggy

**Comment by pshaughnessy88** [comment]
We found the issue and fixed it. We needed to add bs- to the data-toggle and data-target.

**Reply by guustnieuwenhuis** [reply]
Hi,
Was the `bs-` missing in. the Bootstrap 5 theme code, or in your customizations?

**Reply by Tigerlady** [reply]
Guust,  It is missing in the Bootstrap 5 Theme code.  I have also noticed that the shadowbox for images also does not work because Bootstrap changed the CSS markup and older sites that use this get broken upon upgrading.

**Reply by guustnieuwenhuis** [reply]
Thanks! We'll look into this.

**Reply by guustnieuwenhuis** [reply]
The issue has been fixed in MasaBootrap5.

**Comment by pshaughnessy88** [comment]
Thank you so much!

Peggy

From: Guust ***@***.***>
Sent: Friday, September 8, 2023 3:09 AM
To: ***@***.***>
Cc: Shaughnessy, ***@***.***>; ***@***.***>
Subject: [EXTERNAL] Re: [MasaCMS/MasaCMS] Masa Mobile Navigation not displaying (Discussion #234)


The issue has been fixed in MasaBootrap5.
